components/form/field.scss

```diff
@@ -8,12 +8,16 @@
   text-align: left;
 
   &_label {
-    // .is-p is font-properties, defined in typography.
-    @extend .is-p;
-
-    display: inline-block;
-    margin-bottom: 0.2rem;
+    // update to is-p-small, after typo is done, and adjust line height.
+    font-size: 0.875rem;
     font-weight: bold;
+    font-style: normal;
+    color: var(--hdd-color-black);
+    line-height: 16px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.2px;
```

Comment:
Mistenker dette er kopiert rett fra figma. Det tar med mye unødvendig styles som ikke gjør noe og skaper forvirring

Reply:
eller jeg som tar feil :)

Reply:
Noe av det så jeg nå er unødvendig men andre er nødvendige for at den skal fungere korrekt. Er avhengig av noe av stilen her så lenge ikke typgraphien er merget enda. Kan gjerne sette denne på vent inntil den er i orden, så slipper man to gjennomganger.